x/profiles/keeper/grpc_query_test.go

```diff
@@ -514,11 +532,20 @@ func (suite *KeeperTestSuite) Test_UserBlocks() {
 
 	for _, uc := range usecases {
 		uc := uc
+		suite.SetupTest()
 		suite.Run(uc.name, func() {
-			suite.SetupTest()
+			profile := suite.CreateProfileFromAddress(suite.testData.user)
+			otherProfile := suite.CreateProfileFromAddress(suite.testData.otherUser)
+
+			err := suite.k.StoreProfile(suite.ctx, profile)
+			suite.Require().NoError(err)
+
+			err = suite.k.StoreProfile(suite.ctx, otherProfile)
+			suite.Require().NoError(err)
 
 			for _, UserBlock := range uc.storedUserBlocks {
```

Comment:
I think here should be lowercase naming.

Reply:
Fixed